pkgs/development/python-modules/sphinx-autobuild/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
+
+  propagatedBuildInputs = [ sphinx  livereload ];
+
+  meta = with lib; {
+    description = "Rebuild Sphinx documentation on changes, with live-reload in the browser.";
+    homepage = "https://github.com/GaretJax/sphinx_autobuild";
```

Comment:
The link is 404.

```suggestion
    homepage = "https://github.com/executablebooks/sphinx-autobuild";
```

Reply:
I think we should also fetch https://pypi.org/project/sphinx-autobuild/ then.

Reply:
i think we already fetch from that pypi space in this PR, don't we?